core/lib/utils/log_tracking.dart

```diff
@@ -38,15 +38,17 @@ class LogTracking {
   }
 
   Future _executeTraceLog() async {
-    while (true) {
-      if (_messagesQueue.isEmpty) {
-        _isScheduled = false;
-        return;
+    try {
```

Comment:
why not `try-catch` inside `while`, we still good in log

Reply:
done